pre-commit/strip.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# strip all TARGET string what matches PATTERN
+# sh strip.sh abcb b -> ab
+
+if [ $# -ne 2 ]; then
+  echo "2 arguments are required"
+  echo "1st arg is source string, 2nd arg is pattern string"
```

Comment:
Traditionally, this is where a usage statement would go
```suggestion
  echo "usage: $0 'source string' 'pattern string'"
```
Including `$0` helps track down what script this happened in, rather than having to grep for the error message. :)